### PR TITLE
Use genai.protos to access the raw protos.

### PR DESCRIPTION
**Pull request by MarkDaoust**
Use `genai.protos` to access the raw protos.

### CODE REVIEW

File: site/en/gemini-api/docs/function-calling/python.ipynb [hunk 1]
```diff
@@ -131,7 +131,6 @@
         "import time\n",
```
Comment (markmcd) [inline site/en/gemini-api/docs/function-calling/python.ipynb:131]: <blockquote>The&nbsp;<code>google.ai.generativelanguage</code>&nbsp;client library</blockquote><p> </p><p>Are you cleaning these up too? Or just protos / glm directly?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/446/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

File: site/en/gemini-api/docs/semantic_retrieval.ipynb [hunk 2]
```diff
@@ -70,7 +70,7 @@
         "\n",
```
Comment (markmcd) [inline site/en/gemini-api/docs/semantic_retrieval.ipynb:70]: <p>Same here I guess - do we want to talk about "the Generative Language API" ? Understood if it's out of scope for this change, just made me wonder.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/446/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (markmcd) [inline site/en/gemini-api/docs/semantic_retrieval.ipynb:70]: <pre class="ql-syntax" spellcheck="false">Line #8.    request = glm.CreatePermissionRequest(
</pre><p>Still have some glm references in here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/446/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

**Review by markmcd — APPROVED** [review]
LGTM

File: site/en/gemini-api/docs/function-calling/python.ipynb (hunk 1 above)
Comment (MarkDaoust) [reply site/en/gemini-api/docs/function-calling/python.ipynb:131]: <p>Fixed.</p>

File: site/en/gemini-api/docs/semantic_retrieval.ipynb (hunk 2 above)
Comment (MarkDaoust) [reply site/en/gemini-api/docs/semantic_retrieval.ipynb:70]: <p>Done. Gemini API.</p>

Comment (MarkDaoust) [reply site/en/gemini-api/docs/semantic_retrieval.ipynb:70]: <p>Yeah, this doc needs a more extensive replacement. Next PR.</p>

**Pull request merged**